README.md

```diff
@@ -0,0 +1,44 @@
+# CDP Case management plugin
+
+
+## Development
+
+Before running make sure to have the following *environment variables* defined correctly:
+
+- AWS_ACCESS_KEY_ID
+- AWS_SECRET_ACCESS_KEY
+- S3_BUCKET_NAME
+- S3_BUCKET_URL
```

Comment:
I think that this should be derived from bucket name and region

Reply:
This is my first impression, but I'm not sure if this is always the case!

Reply:
What do I do? Infer URL or leave as is?

Reply:
Just infer the URL

Reply:
done!